recipes/sshs/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+schema_version: 1
+
+context:
+  version: "4.7.2"
+
+package:
+  name: sshs
+  version: ${{ version }}
+
+source:
+  url: https://github.com/quantumsheep/sshs/archive/refs/tags/${{ version }}.tar.gz
+  sha256: cb14fd50599bfb8de3f04b00cffb8eac9207f67e0ebbdd9380c311d141882f3b
+
+build:
+  number: 0
+  script:
+    - if: win
+      then:
+        - set CARGO_PROFILE_RELEASE_STRIP=symbols
```

Comment:
Any reason to not set these for non-windows builds? See:
https://conda-forge.org/docs/maintainer/example_recipes/rust/

as an alternative example of this section.

Reply:
@synapticarbors Thank you for the review and the example.These environment variables are also useful for Unix groups to reduce binary package size. I revised the recipe based on your example, and I have confirmed that all checks have passed.